pkg/kubelet/prober/worker.go

```diff
@@ -314,6 +322,29 @@ func (w *worker) doProbe(ctx context.Context) (keepGoing bool) {
 		return true
 	}
 
+	if resultChanged {
+		var eventType string
+		var eventReason string
+		if result == results.Success {
+			eventType = v1.EventTypeNormal
+			eventReason = events.ContainerProbeResultSuccess
+		} else {
+			eventType = v1.EventTypeWarning
+			eventReason = events.ContainerProbeResultFailure
+		}
+		recordContainerEvent(
+			w.recorder,
+			w.pod,
+			&w.container,
+			eventType,
+			eventReason,
+			"%s probe result changed to %s%s",
+			w.probeType,
+			result,
+			getPrefixedOutputEventString(result, output),
```

Comment:
Is this the same output that's already emitted by `prober.probe`? If so, does it really make sense to repeat it here and change a lot of code to achieve this?